src/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixer.php

```diff
@@ -158,10 +159,16 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
                 $content = $this->removeSuperfluousInheritDoc($content);
             }
 
+            $namespaceAnalysis = (new NamespacesAnalyzer())->getNamespaceAt($tokens, $index);
+            $namespace = $namespaceAnalysis->getFullName();
```

Comment:
`$namespaceAnalysis` is used once, so maybe there's no need to create variable.

```suggestion
            $namespace = (new NamespacesAnalyzer())->getNamespaceAt($tokens, $index)->getFullName();
```

Also I would prefer to add `?: null` and get rid of `if` below, but PHPStan's `DisallowedShortTernaryRule` strict rule would complain 😕. I am not a fan of this rule (side note).

Reply:
`?:` is really not good as non-strict comparison is used, NS (PHP identifier in general) cannot start with `0`, thus it seems fine here, but I agree with phpstan and it should be not used nowhere...

Thus I am leaving this code as is, but I would not object any CS change before/after merge as I am not maintainer :)

Reply:
But the point of this comment was removing intermediary `$namespaceAnalysis` which is used only once (so superfluous for me), I mentioned `?:` only as a side note 😉.

Reply:
Well I read it, but not sure if the complex one liners are more readable... But I landed it so this PR has zero feedback left :)